### PR TITLE
Repo standardization

### PR DESCRIPTION
**Pull request by ryanml**
Related: https://github.com/MetaMask/MetaMask-planning/issues/32
- Adding GH actions workflow
- Enforcing minimum Node version of 12
- Adding CODEOWNERS file
- Adding basic .gitattributes file
- Adding basic .editorconfig file

### CODE REVIEW

File: .github/workflows/test.yml
Comment (ryanml) [inline .github/workflows/test.yml:5]: note, default branch name

File: .github/CODEOWNERS
Comment (ryanml) [inline .github/CODEOWNERS:4]: not sure if `Devs` or anyone else should be a codeowner

Comment (rekmarks) [reply .github/CODEOWNERS:4]: We probably should add `Devs` here as well in case we need to make an update.

Comment (ryanml) [reply .github/CODEOWNERS]: Done 👍

**Review by rekmarks — APPROVED** [review]
LGTM!

**Comment by xrn** [comment]
Hi - after merge on master there are problems with build

**Comment by ryanml** [comment]
@danfinlay can you disable CircleCI from running on this repo? Thanks! @xrn - functionally things are fine, we just need to remove the now defunct CircleCI build

**Comment by danfinlay** [comment]
I have stopped the building on CircleCI.

**Pull request merged**
